pkg/render/policyrecommendation.go

```diff
@@ -266,6 +332,15 @@ func allowTigeraPolicyForPolicyRecommendation(cfg *PolicyRecommendationConfigura
 			Destination: networkpolicy.ESGatewayEntityRule,
 		},
 	}
+
+	if !cfg.ManagedCluster {
+		egressRules = append(egressRules, v3.Rule{
```

Comment:
We also need to relax the policy for Linseed.

Reply:
Do we currently have tests for the network policies installed by operator for policy recommendation controller ?

Reply:
I've added tests to policyrecommendation_test.go